Formula/packer.rb

```diff
@@ -22,340 +22,46 @@ class Packer < Formula
-    url "https://github.com/rackspace/gophercloud.git",
-      :revision => "53d1dc4400e1ebcd37a0e01d8c1fe2f4db3b99d2"
-  end
-
-  go_resource "github.com/satori/go.uuid" do
-    url "https://github.com/satori/go.uuid.git",
-      :revision => "d41af8bb6a7704f00bc3b7cba9355ae6a5a80048"
-  end
-
-  go_resource "github.com/tent/http-link-go" do
-    url "https://github.com/tent/http-link-go.git",
-      :revision => "ac974c61c2f990f4115b119354b5e0b47550e888"
-  end
-
-  go_resource "github.com/ugorji/go" do
-    url "https://github.com/ugorji/go.git",
-      :revision => "646ae4a518c1c3be0739df898118d9bccf993858"
-  end
-
-  go_resource "golang.org/x/crypto" do
-    url "https://go.googlesource.com/crypto.git",
-      :revision => "1f22c0103821b9390939b6776727195525381532"
-  end
-
-  go_resource "golang.org/x/net" do
-    url "https://go.googlesource.com/net.git",
-      :revision => "6ccd6698c634f5d835c40c1c31848729e0cecda1"
-  end
-
-  go_resource "golang.org/x/oauth2" do
-    url "https://go.googlesource.com/oauth2.git",
-      :revision => "8a57ed94ffd43444c0879fe75701732a38afc985"
-  end
-
-  go_resource "golang.org/x/sys" do
-    url "https://go.googlesource.com/sys.git",
-      :revision => "50c6bc5e4292a1d4e65c6e9be5f53be28bcbe28e"
-  end
-
-  go_resource "google.golang.org/api" do
-    url "https://github.com/google/google-api-go-client.git",
-      :revision => "ff0a1ff302946b997eb1832381419d1f95143483"
-  end
-
-  go_resource "google.golang.org/appengine" do
-    url "https://github.com/golang/appengine.git",
-      :revision => "6bde959377a90acb53366051d7d587bfd7171354"
-  end
-
-  go_resource "google.golang.org/cloud" do
-    url "https://code.googlesource.com/gocloud.git",
-      :revision => "5a3b06f8b5da3b7c3a93da43163b872c86c509ef"
-  end
-
-  go_resource "gopkg.in/fsnotify.v1" do
-    url "https://gopkg.in/fsnotify.v1.git",
-      :revision => "8611c35ab31c1c28aa903d33cf8b6e44a399b09e"
-  end
-
-  go_resource "gopkg.in/tomb.v1" do
-    url "https://gopkg.in/tomb.v1.git",
-      :revision => "dd632973f1e7218eb1089048e0798ec9ae7dceb8"
-  end
-
-  go_resource "gopkg.in/xmlpath.v2" do
-    url "https://gopkg.in/xmlpath.v2.git",
-      :revision => "860cbeca3ebcc600db0b213c0e83ad6ce91f5739"
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools.git",
+      :revision => "8b3828f1c8f7c67e5ebb863a4c632937778eaaae"
   end
 
   def install
     ENV["XC_OS"] = "darwin"
     ENV["XC_ARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
     ENV["GOPATH"] = buildpath
-    # For the gox buildtool used by packer, which doesn't need to
-    # get installed permanently
+    # For the gox buildtool used by packer, which
+    # doesn't need to be installed permanently.
     ENV.append_path "PATH", buildpath
 
     packerpath = buildpath/"src/github.com/mitchellh/packer"
     packerpath.install Dir["{*,.git}"]
     Language::Go.stage_deps resources, buildpath/"src"
-    mkdir_p buildpath/"bin"
+    (buildpath/"bin").mkpath
```

Comment:
I suspect so, but since we reference `buildpath` everywhere else in this initial block I bowed to consistency, heh.